lib/ruby_lsp/tapioca/addon.rb

```diff
@@ -106,6 +110,42 @@ def workspace_did_change_watched_files(changes)
           constants: constants,
         )
       end
+
+      private
+
+      sig { void }
+      def handle_gemfile_changes
+        return unless File.exist?(".git")
+
+        gemfile_status = %x(git status --porcelain Gemfile.lock).strip
+        return if gemfile_status.empty?
```

Comment:
(🤔 should be continue only if the output begins with `M`, indicating a modification?)